### DISCUSSION TITLE
Question on install

### DISCUSSION THREAD

**Discussion by Runzi333** [wooningeire/plover-touchscreen-stenotype#14] [Q&A]
I've been trying to install this but it doesn't seem to be showing on the plugin manager in Plover, is there another way to install it?

**Comment by wooningeire** [comment]
Currently, the plugin doesn't appear in the plugin manager because it hasn't been entered into the [plugin registry](https://github.com/openstenoproject/plover_plugins_registry) yet (there is an open pull request though). In the meantime, I believe you can install a plugin from a git repository by doing the following:
- If you're on Windows, locate your Plover installation (if it's installed system-wide, it will be at `C:\Program Files\Open Steno Project\Plover 4.0.0rc2`)
- Right-click an empty space in the folder in Windows Explorer and select "Open in Terminal"
- Run the command `plover_console -s plover_plugins install git+https://github.com/wooningeire/plover-touchscreen-stenotype`

**Comment by wooningeire** [comment]
I've updated the package on PyPI, so you can also follow the instructions on https://plover.wiki/index.php/Plugins to download a PyPI package as a plugin
